### ISSUE TITLE
Cursor skin pack

### ISSUE THREAD

**Issue by momo10015** [alexzielenski/Mousecape#128]
Hello, can I put this pack with different cursors to work with mousecape? If you can please tell me how. thanks in advance.

http://www.rw-designer.com/cursor-set/diablo-2-cursor-set-zip

**Comment by NordJan** [comment]
No, at least not directly. The cursor pack contains .ani files, those are Windows animated icon files. If you convert them to .png files, you can build a .mousecape from them.
Cheers. Nord

**Comment by momo10015** [comment]
Hello Nord, Thank you for your answer. So the only thing I have to do is convert every file in the pack to png files? Are there more steps to it and finally where in the mousecape app do I put all these pictures?

I appreciate the help

Kind regards.

**Comment by alexzielenski** [comment]
This answer from @MUI68K has some helpful information regarding this: https://github.com/alexzielenski/Mousecape/issues/125#issuecomment-642927241

Once you have the pngs they can be dragged into the mousecape cursor edit window.

1. Start a new Cape (Command-N)
2. Edit your new cape (Select and press Command-E)
3. Add cursors to your cape (plus in bottom left)
4. Drag PNG images into dashed dropzones

**Comment by momo10015** [comment]
Thank you everyone I managed to get some working but now I have a few more questions. I found this animated cursor (http://www.rw-designer.com/cursor-detail/47799) can I put this as my cursor and how? another question I have is when I hover over something I can type on the cursor changes and I want to put what it changes to. is this possible?

**Comment by MUI68K** [comment]
> Thank you everyone I managed to get some working but now I have a few more questions. I found this animated cursor (http://www.rw-designer.com/cursor-detail/47799) can I put this as my cursor and how? another question I have is when I hover over something I can type on the cursor changes and I want to put what it changes to. is this possible?

Might help you a little - I took your link and used it as an example in the wiki post. I'll update it with a results image in just a short while
https://github.com/alexzielenski/Mousecape/wiki

**Comment by momo10015** [comment]
Okay thank you
